packages/persist/lib/records.ts

```diff
@@ -124,8 +126,8 @@ export async function persistRecords({
         });
         await updateSyncJobResult(syncJobId, updatedResults, baseModel);
 
-        metrics.increment(metrics.Types.PERSIST_RECORDS_COUNT, records.length);
-        metrics.increment(metrics.Types.PERSIST_RECORDS_SIZE_IN_BYTES, recordsSizeInBytes);
+        metrics.increment(metrics.Types.PERSIST_RECORDS_COUNT, records.length, { accountId });
+        metrics.increment(metrics.Types.PERSIST_RECORDS_SIZE_IN_BYTES, recordsSizeInBytes, { accountId });
```

Comment:
This is the metric change, everything else purpose is to make the account available in the endpoint handler logic